.babelrc

```diff
@@ -1,7 +1,19 @@
 {
   "presets": [
     "react",
-    "es2015",
+    ["env", {
+      "targets": {
+        "useBuiltIns": "entry",
+        "browsers": [
```

Comment:
Since we use this [here](https://github.com/cfpb/hmda-platform-ui/pull/793/files#diff-11e9f7f953edc64ba14b0cc350ae7b9dR39) and [here](https://github.com/cfpb/hmda-platform-ui/pull/793/files#diff-5b57b65a004a69d9237f08df4fc77e53R39), should we pull it out into a separate file to make it easier to update?

Reply:
I'm doing this in the nodemon PR, creating a common config that the dev and prod setups inherit